modules/concept-docs/pages/durability-replication-failure-considerations.adoc

```diff
@@ -12,16 +14,24 @@ It is the responsibility of the development team and the software architect to e
 
  
 
-include::7.0@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=intro]
+include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=intro]
+
+include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=syncrep]
+include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=syncrep3]
+
+
+== Older Server Versions
+// include::{version-server}@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=older]
+If a version of Couchbase Server lower than 6.5 is being used then the fallback is 'client verified' durability.
 
-include::7.0@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=syncrep]
-include::7.0@sdk:shared:partial$durability-replication-failure-considerations.adoc[tag=syncrep3]
+WARNING: Client Verified durability is supported in xref:3.2@durability-replication-failure-considerations.adoc#older-server-versions[PHP SDK 3.2] but not in 4.0.
+Legacy support will be available in a later 4.x release.
```

Comment:
Was that a firm commitment?

Reply:
We ran these wordings by SDK & PM on Friday sync for *Node*.
Will signal it for them to review on Staging?